poi/poi_types.xml

```diff
@@ -90,6 +90,8 @@
 	<poi_additional name="operator_type_cooperative" tag="operator:type" value="cooperative"/>
 	<poi_additional name="owner" tag="owner" type="text" order="92"/>
 	<poi_additional name="brand" tag="brand" type="text" order="93"/>
+	<poi_additional name="manufacturer" tag="manufacturer" type="text"/>
+	<poi_additional name="model_type" tag="model" type="text"/>
```

Comment:
There is no `model_type` tag in OSM

Reply:
Hi,
oh, yeah, sure, you're right. I meant the key "model" and gave it the name "model_type", because I though that the **name** "model" was already token in the poi_types.xml for the tag **shop=model**. Buy maybe my thought that a name in the poi_types file isn't allowed to be twiche there was wrong. I can change it to name=model if this would help.

Reply:
You are right, I misunderstood this string